### DISCUSSION TITLE
Data set used by Neural network

### DISCUSSION THREAD

**Discussion by Taraneh1367** [NVlabs/sionna#46] [Q&A] [answered]
Hello,

I was wondering if it is possible to send me the data set you used for the neural receiver's training?

I wanted to ask you if you have any other example that model the LS channel estimator using neural receiver? I need a data set to just model LS Channel estimator using neural network.

Any help from you is really appreciated,

**Comment by jhoydis** [comment, marked as answer]
This question was already answered here: https://github.com/NVlabs/sionna/issues/44
